TIL/topics/02_git/git_basics.md

```
@@ -0,0 +1,90 @@
+- 원격 저장소 정보 확인 및 설정
+
+### git remote -v
+- 원격 저장소 url 포함 정보 보기
+
+### git remote add origin url 
+- 로컬 저장소를 github 에 있는 원격 저장소와 연결
+
+### git remote remove origin
+- 원격 연결 제거(끊기)
+
+### git remote -v 
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)